rules/alerting.go

```diff
@@ -153,6 +155,25 @@ func (r *AlertingRule) sample(alert *Alert, ts time.Time) promql.Sample {
 	return s
 }
 
+// forStateSample returns the sample for ALERTS_FOR_STATE.
+// v should be -1 for StateInactive, UnixNano for StatePending and StateFiring.
```

Comment:
There should be no value for inactive, we don't know the labels of inactive alerts

Reply:
Forgot to remove that comment. Currently staleness handling takes care of it.